### PR TITLE
Fix package metadata

### PR DESCRIPTION
**Pull request by Eilon**
Plus a few other pre-Core names.

cc @kichalla @rowanmiller @divega @ajaybhargavb

### CODE REVIEW

File: src/Microsoft.AspNetCore.Diagnostics.Elm/Views/DetailsPage.cs [hunk 1]
```diff
@@ -423,9 +423,9 @@ public DetailsPage()
 #line hidden
 
             WriteLiteral("                    <option");
-            BeginWriteAttribute("value", " value=\"", 7693, "\"", 7713, 1);
+            BeginWriteAttribute("value", " value=\"", 7715, "\"", 7735, 1);
```
Comment (muratg) [inline src/Microsoft.AspNetCore.Diagnostics.Elm/Views/DetailsPage.cs]: Why did the values change?

File: src/Microsoft.AspNetCore.Diagnostics.Elm/project.json [hunk 2]
```diff
@@ -1,6 +1,8 @@
 {
   "version": "0.1.0-*",
-  "description": "ASP.NET 5 Middleware to capture and display the request logs.",
+  "description": "ASP.NET Core  error logging middleware (ELM) to capture and display request logs.",
```
Comment (muratg) [inline src/Microsoft.AspNetCore.Diagnostics.Elm/project.json]: "Error Logging Framework" instead? (proper noun)

**Comment by muratg** [comment]
Apart from a few comments above, :shipit:

File: src/Microsoft.AspNetCore.Diagnostics.Elm/project.json (hunk 2 above)
Comment (Eilon) [inline src/Microsoft.AspNetCore.Diagnostics.Elm/project.json]: Sure

File: src/Microsoft.AspNetCore.Diagnostics.Elm/Views/DetailsPage.cs (hunk 1 above)
Comment (Eilon) [inline src/Microsoft.AspNetCore.Diagnostics.Elm/Views/DetailsPage.cs]: Probably some change in Razor... probably... :smile:

**Pull request merged**